magda-web-server/src/index.ts

```diff
@@ -225,6 +225,14 @@ topLevelRoutes.forEach(topLevelRoute => {
     });
 });
 
+/**
+ * For whatever reason a user still requests /publishers, redirect to /organisations
+ */
+app.get("/publishers", function(req, res) {
```

Comment:
This kinda works but it'd be even better to have it done at the react-router level: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/Redirect.md.

This way if we missed a `<Link to="/publishers" />` it'll catch that too.